pypuck/pypuck.py

```diff
@@ -77,22 +78,21 @@ def player_stats(start_date=None, end_date=None):
     return pd.DataFrame(page.json()['data'])
 
 
-def attendance(regular=True, playoffs=True, season=None):
+def attendance(regular=True, playoffs=True, start_season=None, end_season=None):
     """
     Query to the Attendance API to get the NHL's seasonal and playoff attendance numbers.
        
-
-
-
     Parameters
     ----------
     regular : boolean (default True).
     
     playoffs : boolean (default True)
```

Comment:
@chuusan I think these should be a little more explicit:

```suggestion
    regular : boolean (default True).
        Whether to query seasonal regular season attendance data.
    playoffs : boolean (default True)
        Whether to query seasonal playoff attendance data.
```